foreman_teamdynamix.gemspec

```diff
@@ -16,4 +16,5 @@ Gem::Specification.new do |s|
   s.test_files = Dir['test/**/*']
   s.add_development_dependency 'rdoc'
   s.add_development_dependency 'rubocop'
+  s.add_dependency 'deface', '>=1.2.0', '< 2.0'
```

Comment:
This dependency version fails to install on our version of Foreman.
Please make your gemspec the same as this one: https://github.com/MiamiOH/foreman_custom_tab/blob/master/foreman_custom_tab.gemspec#L18